galaxy_ng/app/api/ui/viewsets/tags.py

```diff
@@ -21,3 +30,106 @@ def list(self, request, *args, **kwargs):
         serializer = self.get_serializer(page, many=True)
 
         return self.get_paginated_response(serializer.data)
+
+
+class TagFilterOrdering(filters.OrderingFilter):
+    def filter(self, qs, value):
+        if value is not None and any(v in ["count", "-count"] for v in value):
+            order = "-" if "-count" in value else ""
+
+            return qs.filter(
+                ansible_collectionversion__ansible_crossrepositorycollectionversionindex__is_highest=True  # noqa: E501
```

Comment:
does it really need to use the `crossrepo` index? for community we have only one repo and it would be faster to just take the is_highest from the single repo.

Reply:
That's true, but then this wouldn't work for PAH and CRC